Directory.Build.props

```diff
@@ -1,10 +1,13 @@
 <Project>
-  
-  <!-- Common settings for test and product code-->
+
+    <!-- Common settings for test and product code-->
 
   <PropertyGroup>
     <Version>3.0.0</Version>
     <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+
+    <RoslynPackagesVersion>4.0.1</RoslynPackagesVersion>
+    <NuGetPackagesVersion>6.3.1</NuGetPackagesVersion>
```

Comment:
Miscellaneous: defines the packages to use in a single place to make them easier to update.